prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -416,6 +417,11 @@ def prompt_processor() -> Any:
                             LogLevel.ERROR,
                         )
                         structured_output[output[PSKeys.NAME]] = None
+                        if isinstance(
```

Comment:
Do we need this  check for other tools as well?

Reply:
@gaya3-zipstack This main file a part of prompt service. Tool invoke this piece of code